deepcave/evaluators/ablation_importances.py

```diff
@@ -0,0 +1,246 @@
+# noqa: D400
+"""
+# Ablation Importances
+
+This module evaluates the ablation importance.
+
+The ablation method determines the parameter importances between two configurations.
+One being the default configuration and one the incumbent.
+
+## Classes:
+    - AblationImportances: Provide an evaluator of the ablation importances.
+"""
+
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import copy
+
+import numpy as np
+
+from deepcave.constants import COMBINED_COST_NAME
+from deepcave.evaluators.epm.random_forest_surrogate import RandomForestSurrogate
+from deepcave.runs import AbstractRun
+from deepcave.runs.objective import Objective
+
+
+class AblationImportances:
+    """
+    Provide an evaluator of the ablation importances.
+
+    Properties
+    ----------
+    run : AbstractRun
+        The run(s) to analyze.
```

Comment:
Its a late answer, but I think when we pass a group it would be multiple runs

Reply:
Ah yes true, in that case, it would be multiple runs. But as it then is one instance of `Group(AbstractRun)` and we do not pass multiple `AbstractRun` objects, I'd probably still go with "run" :)